ui/helpers/constants/design-system.ts

```diff
@@ -22,12 +22,10 @@ export enum Color {
   primaryAlternative = 'primary-alternative',
   primaryMuted = 'primary-muted',
   primaryInverse = 'primary-inverse',
-  primaryDisabled = 'primary-disabled',
```

Comment:
No instances of `primaryDisabled` used in codebase

![Screenshot 2024-05-31 at 4 29 31 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/1d38d0c3-1517-43c8-82e1-71bc5213b42a)